source/containers.tex

```diff
@@ -8182,11 +8163,11 @@
     const_reference   back() const      { return c.back(); }
     void push(const value_type& x)      { c.push_back(x); }
     void push(value_type&& x)           { c.push_back(std::move(x)); }
-    template <class... Args> void emplace(Args&&... args)
-      { c.emplace_back(std::forward<Args>(args)...); }
+    template <class... Args>
+      void emplace(Args&&... args)      { c.emplace_back(std::forward<Args>(args)...); }
     void pop()                          { c.pop_front(); }
-    void swap(queue& q) noexcept(noexcept(swap(c, q.c)))
-      { using std::swap; swap(c, q.c); }
+    void swap(queue& q)
+      noexcept(noexcept(swap(c, q.c)))  { using std::swap; swap(c, q.c); }
```

Comment:
I agree with the `emplace` change just above, but don't like separating the `noexcept` specification from the rest of the function decl just to line up the braces. Keeping the decl on one line and the body on another makes more sense to me.